.eslintrc

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
Realized the only BabelJS-flavored file we have is the gulp file. Are we planning on switching over the source files to ES6? If not, this does seem a little overkill (even though it is nice) since the majority of our source is CoffeeScript and it be nice to cut down on the dependencies that we need to pull in.